packages/design-system/src/_forms.scss

```diff
@@ -85,3 +85,7 @@ select.form-select,
     }
   }
 }
+
+.form-check-label {
+  user-select: none;
```

Comment:
What is this for? Won't that prevent users from highlighting and selecting text if they want to?

Reply:
Clicking on the label will highlight text.
It felt wrong, and after a bit of searching it looks like common practice to disable text selection on checkbox labels.

Reply:
Interesting, I'm surprised Bootstrap didn't do this by default then. Instead of doing this at the CSS level, where it'll impact every element with the `form-check-label` class, would it be better to use the [`user-select-none`](https://getbootstrap.com/docs/5.3/utilities/interactions/#text-selection) class in the checkbox component?

Reply:
I can do that.
I thought `form-check-label` was only for checkboxes though.
I thought the other fields just had `form-label`.

Reply:
That sounds correct - I just don't want to impact anywhere that might use the `form-check-label` class without using our checkbox implementation. Then again, that could result in different behavior in places like a CMS, provided the `user-select-none` class isn't used as well. With that in mind, I'm more open to keeping this as is. Any more thoughts?

Reply:
Nothing further.
I'm fine with either approach, but I think I prefer styling the existing `form-check-label` class.